### DISCUSSION TITLE
linux中怎么使用？

### DISCUSSION THREAD

**Discussion by swordnine** [1715173329/UnblockNeteaseMusic#444] [Q&A] [answered]
unblockneteasemusic-linux-x64
这个在linux中怎么运行呢？

**Comment by pan93412** [comment, marked as answer]
首先，把這個檔案標成「可執行檔案」

    chmod +x unblockneteasemusic-linux-x64

然後直接執行：

    ./unblockneteasemusic-linux-x64
